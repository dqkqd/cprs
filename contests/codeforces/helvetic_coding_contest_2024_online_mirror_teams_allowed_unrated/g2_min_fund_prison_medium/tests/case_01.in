4
4 6 5
4 3
2 3
2 4
1 2
4 1
3 1
6 6 2
1 4
2 5
3 6
1 5
3 5
6 5
6 5 7
1 4
2 5
3 6
3 5
6 5
7 5 4
1 4
3 6
3 5
6 5
2 7
