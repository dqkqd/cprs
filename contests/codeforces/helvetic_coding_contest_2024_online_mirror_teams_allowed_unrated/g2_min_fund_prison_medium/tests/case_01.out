-1
20
25
33
